src/index.html

```diff
@@ -11,9 +11,55 @@
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Document</title>
   <link rel="stylesheet" href="./style.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap" rel="stylesheet">
 </head>
 <body>
-
+  <div class="card-container" data-qa="card">
+    <div class="item-picture">
+      <a href="#">
+        <img class="picture" src="./images/imac.jpeg" alt="imac" />
```

Comment:
```suggestion
        <img class="picture" src="./images/imac.jpeg" alt="imac photo" />
```

Reply:
Thanks a lot, so I changed 'imac' to 'imac photo' and if I'm right "item-picture" to "item-photon"